scripts/Microsoft.DotNet.Cli.Build.Framework/TargetConditions/EnvironmentAttribute.cs

```diff
@@ -15,6 +15,10 @@ public EnvironmentAttribute(string envVar, params string[] expectedVals)
             {
                 throw new ArgumentNullException("envVar");
             }
+            if (expectedVals == null)
+            {
+                throw new ArgumentNullException("expectedVals");
```

Comment:
The existing pattern was to use a hard coded string, but I would recommend updating the code to use the nameof operator.  Using the nameof operator in this case helps ensure the the correct argument names are used and helps keep the names in sync when refactoring.